drivers/i3c/i3c_cdns.c

```diff
@@ -375,9 +375,10 @@
 #define I3C_MAX_MSGS				10
 #define I3C_SIR_DEFAULT_DA			0x7F
 #define I3C_MAX_IDLE_CANCEL_WAIT_RETRIES	50
-#define I3C_MAX_IDLE_WAIT_RETRIES		5000
 #define I3C_PRESCL_REG_SCALE			(4)
 #define I2C_PRESCL_REG_SCALE			(5)
+#define I3C_IDLE_TIMEOUT_CYC                                                                       \
+	(I3C_WAIT_FOR_IDLE_STATE_US * (sys_clock_hw_cycles_per_sec() / USEC_PER_SEC))
```

Comment:
Is `I3C_WAIT_FOR_IDLE_STATE_US` specified somewhere? I can't seem to find it via grep.

Reply:
yeah... when I fixed the space'ing accidently removed it :(

It's fixed now